tests/gtest/avifincrtest.cc

```diff
@@ -16,86 +15,97 @@ using testing::Bool;
 using testing::Combine;
 using testing::Values;
 
-namespace libavif
-{
-namespace
-{
+namespace libavif {
+namespace {
 
 //------------------------------------------------------------------------------
 
 // Used to pass the data folder path to the GoogleTest suites.
-const char * dataPath = nullptr;
+const char *dataPath = nullptr;
```

Comment:
more common would be 'const char* dataPath'. The tool just took a guess here, but it's inconsistent with the formatting in tests/gtest/avifincrtest_helpers.h for example

Reply:
As a one off you could force the option one way or another, then the tool should keep the files in sync